docs/knowledge_base/notifications.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Notifications
+category: 6255e67693d5200013b1fa41
+hidden: false
+---
+
+# Overview
+
+Phylum supports notification systems using email addresses or webhooks for various platforms including Slack, Microsoft Teams, and Discord. This document guides you through setting up notifications for each of these platforms.
+
+Notifications are a Phylum Pro feature and can be configured at the group or project level. Currently, the only trigger for notifications is [continuous monitoring](https://docs.phylum.io/docs/continuous_monitoring) events.
+
+## Notification Setup
+
+### Email Notifications
+
+1. Navigate to the 'Notifications' section in your Phylum group or project.
+2. Enter the email address where you wish to receive notifications.
+3. Save your settings to enable email notifications.
+
+### Slack Webhook Integration
+
+1. Create a Slack webhook following the guide at [Slack Webhook Setup](https://api.slack.com/messaging/webhooks#getting_started).
+2. In Phylum, navigate to the 'Notifications' section.
+3. Enter your Slack webhook URL.
+4. Perform a test to ensure the notifications are being sent correctly.
+
+### Microsoft Teams Webhook Integration
```

Comment:
These could be condensed into one section with links to setup for the different platforms being sub-items under `1.`

Reply:
@mathew-horner looks updated, is this resolved?